README.md

```diff
@@ -106,8 +106,7 @@ React is a powerful Javascript library used to build more complicated user inter
 Ruby is a backend, object-oriented programming language. Here we will cover the fundamentals of the Ruby language and revisit the fundamental concepts shared by many programming languages.
 
 1. [Ruby Introduction](./ruby/intro.md)
-1. [Ruby Conditionals](./ruby/conditionals.md)
-1. [Ruby Methods](./ruby/methods.md)
+1. [Ruby Conditionals and Methods](./ruby/conditionals.md)
```

Comment:
I never noticed that everything starts with a `1.` Is that how the list gets ordered under the Ruby Section?

Reply:
Markdown does an interesting thing were if you label everything as `1.` it will render them appropriately. That way if you have to change one item in the list you don't have to relabel everything.